samples/rust/Makefile

```diff
@@ -0,0 +1,26 @@
+PROJECT_NAME = xboxrusttest
+XISO_NAME = rusttest
+XBE_TITLE = rusttest
+NXDK_DIR = /home/$(USER)/rust_nxdk
+OUTPUT_DIR = bin
+RUSTFLAGS := --target i686-pc-windows-gnu
```

Comment:
This looks problematic. The GNU ABI drivers differs from the MSVC ABI used by nxdk. Does rust not support MSVC, or is that a limitation of the cross-compiling tool?

Reply:
Oh, Rust does support MSVC, and so does the cross-compiling tool, I just thought it wouldn't make a difference so I prefered to use the GNU toolchain. I'll change that to MSVC, and, well, if we're lucky, perhaps it might even fix some problems.

Reply:
Nevermind, it's actually not supported by the cross-compile tool.

We might have to make a custom Docker image for it (perhaps we could reuse components from the nxdk Docker images, or perhaps even the whole images with just some more stuff?).